KopernicusExamples/Creating New Bodies/GasGiant/GasPlanet.cfg

```diff
@@ -1,9 +1,23 @@
-@Kopernicus:AFTER[Kopernicus]
+@Kopernicus:AFTER[Kopernicus]:FOR[Bastion]
 {
     Body
     {
         name = Bastion
-
+		cacheFile = KopernicusExamples/Creating New Bodies/GasGiant/Cache/Bastion.bin
```

Comment:
I would prefer if the lines you added would be indented the same way the other lines are indented. Just so the configs stay as readable as possible. Either reindent your additions using 4 spaces (if you are using Notepad++ you can enable displaying whitespace characters - that helps a lot), or reindent all the others using tabs. I don't really care.